### PR TITLE
Added left navigation to the fastlane

### PR DESCRIPTION
**Pull request by line47**
[Preview link](https://federalist-proxy.app.cloud.gov/preview/18f/nsf-sbir/fastlane-nav/fastlane/fastlane1/)

Added a left navigation to the fastlane pages. We need to create an additional issue for the design.

### Screenshot 
screen shot 2017-06-16 at 2 54 01 pm

### CODE REVIEW

File: _data/navigation.yml [hunk 1]
```diff
@@ -17,7 +17,17 @@ primary:
   - text: Apply for funding
     href: /apply/
 
-secondary: []
+fastlane:
+  - text: Fastlane Overview
+    href: /fastlane/fastlane1/
```
Comment (gemfarmer) [inline _data/navigation.yml]: to be consistent with the rest of the site, should we maybe do something like `/fastlane/1/` or `/fastlane/fastlane-1/`?

File: _layouts/secondary-narrow.html [hunk 2]
```diff
@@ -4,15 +4,19 @@
 ---
 {% assign sidenav = site.data.navigation[page.sidenav] | default: page.sidenav %}
 {% if sidenav %}
+<div class="usa-grid">
 <aside class="usa-width-one-fourth usa-layout-docs-sidenav">
+  <div class="usa-section">
```
Comment (gemfarmer) [inline _layouts/secondary-narrow.html]: It looks like we are missing an end tag `</div>` here.

File: _sass/side-nav.scss [hunk 3]
```diff
@@ -0,0 +1,42 @@
+aside.usa-layout-docs-sidenav {
+
+  .usa-sidenav-list {
```
Comment (gemfarmer) [inline _sass/side-nav.scss]: Does this class need to be nested within `aside.usa-layout-docs-sidenav`? The specificity is pretty high, and actually looks like it prevents the `usa-active` class from working: 
screen shot 2017-06-20 at 10 19 23 am

File: _sass/three-fourths-tmpl.scss [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  .usa-section.usa-content {
+
+    h1 {
+      letter-spacing: -.015em;  // $kern-tight
+    }
+
+    h2 {
+      font-size: $font-size-m;
+      margin-top: $space-xl;
+      // margin-bottom: $space-m;
```
Comment (gemfarmer) [inline _sass/three-fourths-tmpl.scss]: comment can be removed

File: _data/navigation.yml (hunk 1 above)
Comment (line47) [inline _data/navigation.yml]: I like `/fastlane/fastlane-1/` let's go with that

File: _layouts/secondary-narrow.html [hunk 5]
```diff
@@ -4,15 +4,19 @@
...
   {% include sidenav.html links=sidenav %}
+</div>
```
Comment (line47) [inline _layouts/secondary-narrow.html]: @gemfarmer the end `</div>` is down here I don't think this file is missing any `</div>`

File: _sass/side-nav.scss (hunk 3 above)
Comment (line47) [inline _sass/side-nav.scss]: I think we could just use `.usa-sidenav-list li a` and then modify the `current` state of the nav. 
@femmebot what do you think about that?

File: _sass/three-fourths-tmpl.scss (hunk 4 above)
Comment (line47) [inline _sass/three-fourths-tmpl.scss]: Removed 👍

File: _sass/side-nav.scss (hunk 3 above)
Comment (gemfarmer) [inline _sass/side-nav.scss]: Would just `.usa-sidenav-list a` work? It looks like that is the level of specificity that the USWDS standards uses.

Comment (line47) [inline _sass/side-nav.scss]: Yeah I think so! I'll make the update so we can get this merged 🚀

**Pull request merged**